scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_model.py

```diff
@@ -177,7 +179,7 @@ def do_single_tf_fit(self, parameter_dict):
 
     def do_single_fit_and_return_workspace_parameters_and_fit_function(
             self, parameters_dict):
-        if 'DoublePulseEnabled' in self.context.gui_context and self.context.gui_context['DoublePulseEnabled']:
+        if self.double_pulse_enanbled():
```

Comment:
Sorry to be really picky. There is a typo here that my eyes now can't unsee...

Reply:
thats a good spot. The if statement would not check correctly with that typo